src/cli/ionic-config.ts

```diff
@@ -0,0 +1,45 @@
+import { readJSON, writeJSON, mkdirp } from '@ionic/utils-fs';
+
+import * as os from 'os';
+import * as path from 'path';
+
+import { uuidv4 } from './telemetry/helpers';
+
+export const CONFIG_FILE = 'config.json';
+export const DEFAULT_CONFIG_DIRECTORY = path.resolve(os.homedir(), '.ionic', CONFIG_FILE);
+
+export interface SystemConfig {
+	"telemetry"?: boolean,
```

Comment:
For my own edification, is this to enable/disable telemetry? Is this going to be shared across multiple project types?

Reply:
I'm adding one new prop to the ionic file - that's `telemetry.stencil` - so that a customer could enable telemetry for ionic CLI and disable telemetry for Stencil CLI. Originally when building this interface, I considered overwriting this key, or inferring the value for the `telemetry.stencil` key when first adding it. I went with a simpler approach where stencil only manages its prop, so I will nix the `telemetry` prop in this interface as its unused.